data/transactions/logic/TEAL_opcodes.md

```diff
@@ -854,6 +854,17 @@ When A is a uint64, index 0 is the least significant bit. Setting bit 3 to 1 on
 - pop a byte-array A and integer B. Extract a range of bytes from A starting at B up to but not including B+8, convert bytes as big endian and push the uint64 result. If B+8 is larger than the array length, the program fails
 - LogicSigVersion >= 5
 
+## base64_decode e
+
+- Opcode: 0x5c {uint8 alphabet index}
+- Pops: *... stack*, []byte
+- Pushes: []byte
+- decode X which was base64-encoded using _encoding alphabet_ E. Fail if X is not base64 encoded with alphabet E
+- **Cost**: 25
+- LogicSigVersion >= 6
+
+decodes X using the base64 encoding alphabet E. Specify the alphabet with an immediate arg either as URL and Filename Safe (`URLAlph`) or Standard (`StdAlph`). See <a href="https://rfc-editor.org/rfc/rfc4648.html#section-4">RFC 4648</a> (sections 4 and 5)
```

Comment:
URLEncoding and StdEncoding
?

Reply:
If you think it's worth it -it's a bit of a mouthful, but clearer, and so I'm leaning to your suggestion (even if it's a bit more of a headache to fix). So the TEAL code would look like:

```
...
base64_decode URLEncoding
...
```